### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by rohithasrk**
Updated README with all the instructions and necessary details of the project.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -71,26 +61,88 @@ If you want to contribute, install your cloned fork:
...
+        'django.contrib.messages',
+        'django.contrib.staticfiles',
+        # all-auth
+        'django.contrib.sites',
+        'allauth',
+        'allauth.account',
+        'allauth.socialaccount',
+        # openwisp2 modules
+        'openwisp_users',
         'openwisp_network_topology',
```
Comment (nemesisdesign) [inline README.rst]: ?

Comment (rohithasrk) [inline README.rst]: Anything wrong @nemesisdesign?

File: README.rst [hunk 2]
```diff
@@ -71,26 +61,88 @@ If you want to contribute, install your cloned fork:
...
+        # admin
+        'openwisp_network_topology.admin_theme',
```
Comment (nemesisdesign) [inline README.rst]: sorry, I meant to highlight this line

Comment (rohithasrk) [inline README.rst]: Oh. Well, I thought we'd be adding it too. As there is a PR to add admin theme. #8

**Pull request merged**